src/Razor/src/rzls/rzls.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
This is a deployable asset that goes into the .NET 7 and 8 SDK. As such we need to keep a `net7.0` TFM so it can deploy there. Believe the right change here is the following: 

```suggestion
    <TargetFrameworks>$(DefaultNetCoreTargetFrameworks)</TargetFrameworks>
```

Reply:
That unfortunately doesn't work - since this project is published, using more than one TFM fails the publish. Or at least I didn't figure out how to tell the publishing that it should use just one TFM, but I could try investigating more.

Reply:
You can change publish to only publish a subset of TFMS. Where do you get this error? Should be able to help you set the right properties once I see where the error is coming from.

Reply:
In the CI build of the first commit, it's this error:

<samp>
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [D:\a\_work\1\s\src\Razor\src\rzls\rzls.csproj]
</samp>

Reply:
I tried some MSBuild magic that seems to have fixed it (see the last commit).